docs/ota-client-guide/modules/ROOT/pages/customise-targets-metadata.adoc

```diff
@@ -9,35 +9,45 @@ endif::[]
 
 include::partial$aktualizr-version.adoc[]
 
-In some cases, you might find it useful to include extra metadata about your software inside the signed Uptane metadata that OTA Connect delivers to your device. Some reasons you might want to do this include:
+You may need to include extra metadata about your software inside the signed Uptane metadata that OTA Connect delivers to your device. Some reasons you might want to do this include:
 
-* to provide installation instructions or scripts for an image that can't or shouldn't be included in the image itself
-* to add extra tracking fields for internal compliance or auditing
+* To provide installation instructions or scripts for an image that cannot or should not be included in the image itself.
+* To add extra tracking fields for internal compliance or auditing.
 
 To accommodate this use case, you can manually manage your Uptane metadata and add custom fields to your `targets.json` file.
 
-[NOTE]
-====
-For more information on additional use cases which will require customization of the Targets metadata, please refer to the section https://uptane.github.io/papers/ieee-isto-6100.1.0.0.uptane-standard.html#rfc.section.5.2.3.1.1['Custom metadata about images' in the Uptane standard].
-====
+NOTE: For more information on additional use cases that require customization of the targets metadata, see the https://uptane.github.io/papers/ieee-isto-6100.1.0.0.uptane-standard.html#rfc.section.5.2.3.1.1[related section] of the Uptane standard.
```

Comment:
Why do we link to the 1.0.0 release? Can we not use the latest or at least the 1.0.1 release?

Reply:
It's the link that's mentioned on their documentation portal.
https://uptane.github.io/
Unfortunately, I haven't found the same standard for the 1.0.1 release.

Reply:
https://uptane.github.io/papers/uptane-standard.1.0.1.html

Reply:
Thanks! Changed.